src/components/Dashboard/TodoList.vue

```diff
@@ -1176,10 +1191,6 @@ export default class TodoList extends Vue {
       const paymentStatusCode = header.paymentStatusCode
       const payErrorObj = paymentStatusCode && await PayServices.getPayErrorObj(paymentStatusCode)
 
-      const draft = changeOfRegistration
-      // FUTURE: verify that this checks for all possible data:
-      const haveData = Boolean(draft?.offices || draft?.contactPoint || draft?.parties)
```

Comment:
This and code below doesn't apply to Change of Registration, etc.

`isEmptyFiling` only applies to Incorporation Applications and Registrations (which are used to bootstrap a business) to vary some functionality depending on whether the user has entered any data.